tests/src/test/java/com/datastax/oss/pulsar/functions/transforms/tests/AbstractDockerTest.java

```diff
@@ -105,6 +107,70 @@ public void testPrimitive() throws Exception {
     assertEquals(value.getNativeObject(), "42");
   }
 
+  @Test
+  public void test1() throws Exception {
```

Comment:
CI is failing on this.

Reply:
Oups. This should not have been committed.